### ISSUE TITLE
Support for GHC 8.8 and 8.10

### ISSUE THREAD

**Issue by sjakobi** [hackage-trustees/hackage-cli#15]
I haven't been able to build the current `master` with GHC 8.8 or 8.10, so I ended up using GHC 8.6.

A naive `cabal install --allow-newer -w ghc-8.10.2` fails with

```
lib/Distribution/Server/Util/CabalRevisions.hs:40:1: error:
    Could not find module ‘Distribution.Parsec.Common’
    Perhaps you meant
      Distribution.Parsec.Error (from Cabal-3.2.0.0)
      Distribution.Parsec.Position (from Cabal-3.2.0.0)
      Distribution.Parsec.Warning (from Cabal-3.2.0.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
40 | import Distribution.Parsec.Common (showPWarning, showPError, PWarning (..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

lib/Distribution/Server/Util/CabalRevisions.hs:41:27: error:
    Module ‘Distribution.Text’ does not export ‘Text(..)’
   |
41 | import Distribution.Text (Text(..))
   |                           ^^^^^^^^
```

This is probably related to #13.

**Comment by andreasabel** [comment]
I get the same error on GHC 8.4.4:
```
$ cabal install
Wrote tarball sdist to
/Users/abel/bin/src/hackage-cli/dist-newstyle/sdist/hackage-cli-0.1.0.0.tar.gz
Resolving dependencies...
Build profile: -w ghc-8.4.4 -O1
In order, the following will be built (use -v for more details):
 - Cabal-3.2.1.0 (lib) (requires build)
 - StateVar-1.2.1 (lib) (requires build)
 - blaze-builder-0.4.1.0 (lib) (requires build)
 - base-compat-0.11.2 (lib) (requires build)
 - base-orphans-0.8.4 (lib) (requires build)
 - base64-bytestring-1.2.0.1 (lib) (requires build)
 - bytestring-builder-0.10.8.2.0 (lib) (requires build)
 - call-stack-0.2.0 (lib) (requires build)
 - colour-2.3.5 (lib) (requires build)
 - dlist-1.0 (lib) (requires build)
 - data-default-class-0.1.2.0 (lib:data-default-class) (requires build)
 - fmlist-0.9.4 (lib) (requires build)
 - hsc2hs-0.68.7 (exe:hsc2hs) (requires build)
 - hashable-1.3.0.0 (lib) (requires build)
 - old-locale-1.0.0.7 (lib) (requires build)
 - indexed-traversable-0.1.1 (lib) (requires build)
 - integer-logarithms-1.0.3.1 (lib) (requires build)
 - netrc-0.2.0.0 (lib) (requires build)
 - network-uri-2.6.3.0 (lib) (requires build)
 - parallel-3.2.2.0 (lib) (requires build)
 - primitive-0.7.1.0 (lib) (requires build)
 - reflection-2.1.6 (lib) (requires build)
 - semigroups-0.19.1 (lib) (requires build)
 - split-0.2.3.4 (lib) (requires build)
 - splitmix-0.1.0.3 (lib) (requires build)
 - stringsearch-0.3.6.6 (lib:stringsearch) (requires build)
 - utf8-string-1.0.2 (lib) (requires build)
 - tagged-0.8.6.1 (lib) (requires build)
 - tagsoup-0.14.8 (lib) (requires build)
 - th-abstraction-0.4.2.0 (lib) (requires build)
 - tar-0.5.1.1 (lib) (requires build)
 - transformers-compat-0.6.6 (lib) (requires build)
 - void-0.7.3 (lib) (requires build)
 - exceptions-0.10.4 (lib) (requires build)
 - zlib-0.6.2.2 (lib) (requires build)
 - hackage-cli-0.1.0.0 (lib:cabal-revisions) (requires build)
 - cabal-doctest-1.0.8 (lib) (requires build)
 - contravariant-1.5.3 (lib) (requires build)
 - time-compat-1.9.5 (lib) (requires build)
 - ansi-terminal-0.11 (lib) (requires build)
 - data-default-instances-dlist-0.0.1 (lib:data-default-instances-dlist) (requires build)
 - data-default-instances-containers-0.0.1 (lib:data-default-instances-containers) (requires build)
 - network-3.1.2.1 (lib:network) (requires build)
 - unordered-containers-0.2.13.0 (lib) (requires build)
 - data-fix-0.3.0 (lib) (requires build)
 - case-insensitive-1.2.1.0 (lib) (requires build)
 - data-default-instances-old-locale-0.0.1 (lib:data-default-instances-old-locale) (requires build)
 - vector-0.12.1.2 (lib) (requires build)
 - scientific-0.3.6.2 (lib) (requires build)
 - random-1.2.0 (lib) (requires build)
 - distributive-0.6.2.1 (lib) (requires build)
 - generic-deriving-1.14 (lib) (requires build)
 - transformers-base-0.4.5.2 (lib) (requires build)
 - zlib-bindings-0.1.1.5 (lib) (requires build)
 - base-compat-batteries-0.11.2 (lib) (requires build)
 - ansi-wl-pprint-0.6.9 (lib) (requires build)
 - HsOpenSSL-0.11.5.1 (lib:HsOpenSSL) (requires build)
 - data-default-0.7.1.1 (lib:data-default) (requires build)
 - ListLike-4.7.4 (lib) (requires build)
 - attoparsec-0.13.2.4 (lib) (requires build)
 - uuid-types-1.0.3 (lib) (requires build)
 - comonad-5.0.8 (lib) (requires build)
 - optparse-applicative-0.16.1.0 (lib) (requires build)
 - process-extras-0.7.4 (lib) (requires build)
 - io-streams-1.5.2.0 (lib) (requires build)
 - bifunctors-5.5.9 (lib) (requires build)
 - openssl-streams-1.2.3.0 (lib) (requires build)
 - assoc-1.0.2 (lib) (requires build)
 - semigroupoids-5.3.5 (lib) (requires build)
 - profunctors-5.6.1 (lib) (requires build)
 - http-io-streams-0.1.0.0 (lib) (requires build)
 - these-1.1.1.1 (lib) (requires build)
 - invariant-0.5.4 (lib) (requires build)
 - free-5.1.6 (lib) (requires build)
 - strict-0.4.0.1 (lib) (requires build)
 - adjunctions-4.4 (lib) (requires build)
 - aeson-1.5.5.1 (lib) (requires build)
 - kan-extensions-5.2.1 (lib) (requires build)
 - lens-4.19.2 (lib:lens) (requires build)
 - hackage-cli-0.1.0.0 (exe:hackage-cli) (requires build)
Configuring library for Cabal-3.2.1.0..
clang: warning: argument unused during compilation: '-nopie' [-Wunused-command-line-argument]
Preprocessing library for Cabal-3.2.1.0..
Building library for Cabal-3.2.1.0..
...
Building library 'cabal-revisions' for hackage-cli-0.1.0.0..
[1 of 1] Compiling Distribution.Server.Util.CabalRevisions ( lib/Distribution/Server/Util/CabalRevisions.hs, dist/build/cabal-revisions/Distribution/Server/Util/CabalRevisions.o )

lib/Distribution/Server/Util/CabalRevisions.hs:40:1: error:
    Could not find module ‘Distribution.Parsec.Common’
    Perhaps you meant
      Distribution.Parsec.Error (from Cabal-3.2.1.0)
      Distribution.Parsec.Position (from Cabal-3.2.1.0)
      Distribution.Parsec.Warning (from Cabal-3.2.1.0)
    Use -v to see a list of the files searched for.
   |
40 | import Distribution.Parsec.Common (showPWarning, showPError, PWarning (..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

lib/Distribution/Server/Util/CabalRevisions.hs:41:27: error:
    Module ‘Distribution.Text’ does not export ‘Text(..)’
   |
41 | import Distribution.Text (Text(..))
   |                           ^^^^^^^^
```

**Comment by sjakobi** [comment]
Nice! Thank you, @andreasabel! :)
